lighthouse-core/audits/seo/link-text.js

```diff
@@ -18,6 +18,24 @@ const BLOCKLIST = new Set([
   'more',
   'learn more',
 ]);
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that tells the user their links have descriptive enough text for a search engine crawler to parse them. This is displayed in a list of audit titles that Lighthouse generates. */
+  title: 'Links have descriptive text',
+  /** Title of a Lighthouse audit that tells the user their links have descriptive enough text for a search engine crawler to parse them. This title is shown when there are links on the page without proper text to describe them. */
```

Comment:
I'm not sure if this pattern is followed elsewhere, but I personally find it a tad unclear what's going on after the first sentence since they're the same.

IMO we could flip the first sentence `Title of a Lighthouse audit that tells the user their links **do not** have descriptive enough text for a search engine crawler to parse them.` and then we don't need the 2nd or the 2nd could explain that it's showed in a list of other failing audit titles that lighthouse generates?

Reply:
I guess it depends on how you interpret the `that` clause, if it modifies `audit` I get it, if it modifies `Title` then it's not true :)

Reply:
We do this a lot where the first sentence is shared between the `title` and `failureTitle`, but other places we don't.  When I initially looked I thought this was the standard, but it seems mixed.  So I'm not sure if we want to standardize one way or the other, or move forward this was or that?

https://github.com/GoogleChrome/lighthouse/blob/4f16a6a0354d7d97d055d045aa75f508b5b317e2/lighthouse-core/audits/dobetterweb/dom-size.js#L24-L27

https://github.com/GoogleChrome/lighthouse/blob/4f16a6a0354d7d97d055d045aa75f508b5b317e2/lighthouse-core/audits/time-to-first-byte.js#L13-L16

https://github.com/GoogleChrome/lighthouse/blob/4f16a6a0354d7d97d055d045aa75f508b5b317e2/lighthouse-core/audits/bootup-time.js#L16-L19

Reply:
Took another shot at it with the `that provides detail on` styling.  I think it is much cleaner!